services/workflows-service/src/errors.ts

```diff
@@ -75,6 +76,15 @@ export class ValidationError extends common.BadRequestException {
       path: zodIssue.path.join('.'), // Backwards compatibility - Legacy code message excepts array
     }));
 
-    return new ValidationError(errors);
+    return new BadValidationException(errors);
+  }
+
+  static fromClassValidator(error: ValidationError[]) {
```

Comment:
Please consider checking out of dev for a moment and making unrelated changes on a separate PR. 🙏🏼

Reply:
its related to my pr when i couldnt see validation errors, the api didnt work as expected

Reply:
You're fixing class validator's errors. Its affecting your PR but its unrelated.